internal/repository/integration/position_test.go

```diff
@@ -52,19 +55,41 @@ func DeleteData(posRepo domain.PositionsRepository, positions []domain.Position)
 	return errs
 }
 
-func TestPositionRepository_Create(t *testing.T) {
+func InitPosRepo() (domain.PositionsRepository, error) {
 	config, err := conf.NewConfig()
 	if err != nil {
 		log.Fatalf("Error while getting config: %s", err)
 	}
 
-	db, err := database.ConnectPostgres(config.DB)
-	if err != nil {
-		log.Fatalf("Connection to database failed: %s", err)
+	var positionRepo domain.PositionsRepository
+
+	switch config.Name {
+	case "testpostgres":
```

Comment:
naming should be consistent. In another test value was "postgres". So it should be either "postgres" or "testpostgres" consistently throughout usage.